typesense.config.json

```diff
@@ -2,7 +2,7 @@
     "collections": [
         {
             "name": "docs_main_v1",
```

Comment:
we probably want to point this at the Alias name, but the docs API Key isn't configured for that at the moment.